plugins/node/opentelemetry-instrumentation-ioredis/test/ioredis.test.ts

```diff
@@ -336,16 +336,12 @@ describe('ioredis', () => {
         context.with(trace.setSpan(context.active(), span), () => {
           const stream = client.scanStream();
           stream
-            .on('data', resultKeys => {
-              // `resultKeys` is an array of strings representing key names.
+            .on('data', _resultKeys => {
```

Comment:
Is there a reason to keep the `data` listener at all?

Reply:
registering `data` listener puts the stream into flowing mode. An alternative would be to call `read` and consume the data or call `stream.resume()`.

Reply:
Right. I'd prefer we use that instead of the empty listener actually. Other than that :+1:

Reply:
@Rauno56 
Pushed another commit, now it's using `stream.resume()` instead of the empty listener